project-scratch-def.schema.json

```diff
@@ -216,18 +266,25 @@
             "title": "SubPerStreamingChannel:<value>",
             "pattern": "^(SubPerStreamingChannel\\:[0-9]+$)"
           },
+          {
+            "type": "string",
+            "title": "SubPerStreamingTopic:<value>",
+            "pattern": "^(SubPerStreamingTopic\\:[0-9]+$)"
+          },
           {
             "type": "string",
             "enum": [
               "API",
               "AnalyticsAdminPerms",
-              "AnalyticsAppEmbedded",
```

Comment:
@lcampos - I think `AnalyticsAppEmbedded` is a valid scratch org feature (right, @reidaelliott?), but it might not have gotten added to https://developer.salesforce.com/docs/get_document/atlas.en-us.sfdx_dev.meta (@mblumreich?)

Reply:
After looking through the docs, it looks like that feature needs to go here: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file_config_values.htm. I'll create a work item for 230/patch and get it added.

Reply:
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008xFK2IAM/view

Reply:
@smithgp @mblumreich thanks for catching that, I'll manually add this back for now. Do we have a timeline on when the doc fix will be published ? I'm working on follow up PRs to get this automated and running on a scheduled basis.

Reply:
@lcampos I'll get in for the 230.13 deadline this Thursday